stix_shifter_modules/okta/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,318 @@
+{
+  "eventType": {
+    "key": "x-oca-event.action",
+    "object": "event"
+  },
+  "uuid": {
+    "key": "x-oca-event.extensions.x-okta-event.event_unique_id",
+    "object": "event"
+  },
+  "outcome": {
+    "result": {
+      "key": "x-oca-event.outcome",
+      "object": "event"
+    },
+    "reason": {
+      "key": "x-oca-event.extensions.x-okta-event.outcome_reason",
+      "object": "event"
+    }
+  },
+  "legacyEventType": {
+    "key": "x-oca-event.extensions.x-okta-event.legacy_event_type",
```

Comment:
The extension mappings looks odd. we need to rethink how we should map the extensions. For this kind of mapping, we can simply add custom properties rather than putting in the extension.